MATLAB.gitlab-ci.yml

```diff
@@ -110,3 +111,23 @@ test_artifacts:
 #
 # [8] https://www.mathworks.com/help/matlab/ref/matlab.unittest.plugins-package.html
 # [9] https://www.mathworks.com/help/matlab/matlab_prog/generate-artifacts-using-matlab-unit-test-plugins.html
+
+# Starting in R2022b, the `build` job runs a build using the MATLAB build tool. You can use this job to run the 
+# tasks specified in a file named `buildfile.m` in the root of your repository. To use the job in your pipeline, 
```

Comment:
Replace the sentence:

You can use this job to run the tasks specified in a file named `buildfile.m` in the root of your repository.

with the following:

When you use the `build` job, a file named `buildfile.m` must be in the root of your repository. You can use this job to run the tasks specified in `buildfile.m`.